packages/app/src/actions/index.js

```diff
@@ -81,7 +84,7 @@ export const tDoLoadVisualization = ({
     }
 
     try {
-        return onSuccess(await apiFetchVisualization(type, id))
+        return onSuccess(await apiFetchVisualization(engine, type, id))
     } catch (error) {
```

Comment:
This should catch the 404 error, does it not work?

Reply:
Yes, the problem is with the error passed from the fetch.
The components expect a string, but the error passed is an object.
Fixed.